sc-kpm/sc-agents-common/utils/CommonUtils.cpp

```diff
@@ -63,16 +76,28 @@ string CommonUtils::readString(ScMemoryContext * ms_context, const ScAddr & scLi
 }
 
 string CommonUtils::getIdtfValue(ScMemoryContext * ms_context, const ScAddr & node, const ScAddr & idtfRelation)
+{
+  return getIdtf(ms_context, node, idtfRelation);
+}
+
+string CommonUtils::getIdtf(ScMemoryContext * ms_context, const ScAddr & node, const ScAddr & idtfRelation)
```

Comment:
Let's discuss about so many method renames. We need to avoid too many methods that do the same things and use rename only in significant cases

Reply:
Early this method had name "getIdtfValue". I tink there is sense to use method with name without "value", because I don't understand "identifier value". Casual identifier is a value, it hasn't value

Reply:
May be I'll recommend to remove method and use "getAnyByOutRelation" instead of